docs/_elements/form-controls.md

```diff
@@ -135,15 +135,15 @@ lead: Form controls allow users to enter information into a page.
 
     <ul class="usa-unstyled-list">
       <li>
-        <input id="apple-pie" type="checkbox" name="apple-pie" value="apple-pie" checked />
+        <input id="apple-pie" type="checkbox" name="name[]" value="apple-pie" checked />
```

Comment:
No, actually it's a name "Sojourner Truth." We are not displaying pies, it changed but the internal values didn't change along with it.